impl/client/src/test/javascript/util/xss.spec.js

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright 2010 - 2024 Hitachi Vantara.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ define("common-ui/util/xss",  function(XSS) {
+
+         describe("sanitizeHtml", function() {
+
+             // Test for sanitizing unsafe HTML
+             it("should return correctly formatted locale", function() {
+                 const unsafeHtml = '<div>Hello</div><script>alert("xss")</script>';
+                 const expectedSanitizedHtml = '<div>Hello</div>'; // The <script> tag should be removed
+                 const result = XSS.sanitizeHtml(unsafeHtml);
+                 expect(result).toBe(expectedSanitizedHtml);
+               });
+
+              // Test for safe HTML (should remain unchanged)
+              it('should return safe HTML unchanged', () => {
```

Comment:
Should be preferably using double quotes, only using single quotes to enclose a string if e.g. the string contains a double quote character.

Reply:
This comment applies to all strings in the file, not just in `it` or `describe`.